src/LibraryInstaller.php

```diff
@@ -25,10 +25,12 @@ public function getInstallPath(PackageInterface $package)
         
         if ($packageName == 'moxiemanager') {
             $parentProject = basename(dirname(__DIR__,4 ));
-            if (strpos($parentProject, 'rocket') !== false) {
-                return $this->vendorDir .'/../builtin/public/js/tiny_mce4/plugins/moxiemanager';
+
+            if (strpos(strtolower($parentProject), 'clxfrontendcms') !== false) {
+                return $this->vendorDir .'/../public/js/tinymce/plugins/moxiemanager';
             }
-            return $this->vendorDir .'/plugins/rocket/builtin/public/js/tiny_mce4/plugins/moxiemanager';
+
+            return $this->vendorDir .'/plugins/clxFrontendCms/public/js/tinymce/plugins/moxiemanager';
```

Comment:
@clxmstaab 
Ich bin mir nicht sicher ob es `clxFrontendCms` oder `clxfrontendcms` lauten muss.
Ich vermute `clxFrontendCms` da es hier ja um den physischen Pfad geht.

Um die Installation zu testen muss ich wohl etwas umständlich ein test-Rocket mit dem `complex-library-installer` Featurebranch ausstatten?

Reply:
`clxFrontendCms` ist korrekt analog der login in zeile 36 ff

Reply:
Ok, mergen darfst wohl nur du. Feel free - dann kannst du Release 2.0 erstellen - das häng ich dann in meinen Rocket PR